boards/tsi/ek_tsi_skyp/ek_tsi_skyp-common.dtsi

```diff
@@ -0,0 +1,176 @@
+/*
+ * Copyright (c) 2019-2021 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+ sysclk: system-clock {
+	compatible = "fixed-clock";
+	clock-frequency = <25000000>;
+	#clock-cells = <0>;
+};
+
+gpio0: gpio@9000010 {
+	compatible = "arm,cmsdk-gpio";
+	reg = <0x9000010 0x4>;
+	interrupts = <69 3>;
+	gpio-controller;
+	#gpio-cells = <2>;
+};
+
+gpio1: gpio@9000014 {
+	compatible = "arm,cmsdk-gpio";
+	reg = <0x9000014 0x4>;
+	interrupts = <70 3>;
+	gpio-controller;
+	#gpio-cells = <2>;
+};
+
+gpio2: gpio@9000018 {
+	compatible = "arm,cmsdk-gpio";
+	reg = <0x9000018 0x4>;
+	interrupts = <71 3>;
+	gpio-controller;
+	#gpio-cells = <2>;
+};
+
+gpio3: gpio@900001c {
+	compatible = "arm,cmsdk-gpio";
+	reg = <0x900001c 0x4>;
+	interrupts = <72 3>;
+	gpio-controller;
+	#gpio-cells = <2>;
+};
+
+
+/*eth0: eth@1400000 {*/
+	/* Linux has "smsc,lan9115" */
+	/*compatible = "smsc,lan9220";*/
+	/* Actual reg range is ~0x200 */
+	/*reg = <0x1400000 0x100000>;*/
+	/*interrupts = <49 3>;*/
+/*};*/
```

Comment:
Since we are not using the eth0 yet and maybe put a line saying currently disabled.

Reply:
I will add comments for eth0 and other peripherals.